### DISCUSSION TITLE
Github is not a Social Media Platform

### DISCUSSION THREAD

**Discussion by ppriyank** [community/community#53726] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

The dashboard is not a newsfeed!!

What is the use case of this new dashboard/newsfeed? 
It's not social media that I need to keep updated with what everyone's doing... I don't care who contributes to what or publishes which new repository. It's just noise and random facts.  

Please remove it, when it is still in its beta stage. My humble request...

**Comment by sjaks** [comment]
I think both tabs are useful and have their perks. I like to see what the developers I follows are up to but granted, occasionally I just want to see what is happening in repositories or organizations I'm watching. That's when I use the Following tab.

**Comment by DeerTears** [comment]
I agree, I find it really useful for the people I'm following specifically, like when they start a new repo or if they endorse a repo of decent quality worth my time. Yeah I don't need it but it was making me happy with the stuff I found on github. The unintrusive "recommended for you" repos on the sidebar was fine and I'd rather be able to browse that separately.

I don't mind a feature that you can fully ignore if you don't want to use it, like notifications. But this is ruining the RSS feed-like curation that I found really useful on Github. It feels wrong to water it down into a LinkedIn/Facebook-like "follow for follow" social media network. Most users I'm recommended to follow are former colleagues who haven't even pushed anything this year.

**Comment by Alhadis** [comment]
I wholeheartedly agree with @ppriyank; I come to GitHub to _get away_ from the chronic distractions of social media. This site already has a [trending] page, an [_Explore_] page with similarly-curated "recommendations", and an always-present dashboard sidebar which lists 3 random projects based on whatever I've been doing on GitHub most recently. This is already plenty; adding a tab front-and-centre on my dashboard where I go to check updates from users I've **chosen** to follow feels indicative of an underlying effort to push GitHub further and further towards the domain of social media.

Now, regardless of how you feel towards social media, we can all agree that developers—more so than most people—don't need hand-holding to find projects of interest or like-minded devs. Apart from the [trending] and [_Explore_] pages mentioned above, there's also Reddit, Discord, and hell, even YouTube. Please, let's keep GitHub focussed on what it excels at doing—keeping users productive on their *existing* projects and collaborations.

[trending]:  https://github.com/trending
[_Explore_]: https://github.com/explore

**Comment by ppriyank** [comment]
I would like one more thing. 

Maybe I'm using GitHub correctly, but I'm not scrolling down GitHub to see what's exciting going on out there in the world.  If I  like some work, I search Google / GitHub for its code, and that is it. It's a very targeted search. The dashboard for me is only intended to inform me if my collaborators have pushed some changes to my repository, if someone has raised any issue, etc. Unnecessary information will just dilute this necessary information.
